data/saas/config/hubspot_config.yml

```diff
@@ -0,0 +1,191 @@
+saas_config:
+  fides_key: hubspot_connector_example
+  name: Hubspot SaaS Config
+  description: A sample schema representing the Hubspot connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: hapikey
+
+  client_config:
+    protocol: https
+    host:
+      connector_param: domain
+    authentication:
+      strategy: query_param
+      configuration:
+        token:
+          connector_param: hapikey
+
+  test_request:
+    method: GET
+    path: /companies/v2/companies/paged
+
+  endpoints:
+    - name: contacts
+      requests:
+        read:
+          path: /crm/v3/objects/contacts/search
+          method: POST
+          body: '{
+            "filterGroups": [{
+              "filters": [{
+                "value": "<email>",
+                "propertyName": "email",
+                "operator": "EQ"
+              }]
+            }]
+          }'
+          query_params:
+            - name: limit
+              value: 100
+          param_values:
+            - name: email
+              identity: email
+          postprocessors:
+            - strategy: unwrap
+              configuration:
+                data_path: results
+          pagination:
+            strategy: link
+            configuration:
+              source: body
+              path: paging.next.link
+        update:
+          path: /crm/v3/objects/contacts/<contactId>
+          method: PATCH
+          body: '{
+            "properties": {
+                <masked_object_fields>
+            }
+          }'
+          param_values:
+            - name: contactId
+              references:
+                - dataset: hubspot_connector_example
+                  field: contacts_search.id
+                  direction: from
+    - name: owners
+      requests:
+        read:
+          path: /crm/v3/owners
```

Comment:
Are we still waiting on data for the owners endpoint?

Reply:
Right, it's an empty array, regardless of what data we add in the Hubspot interface, so we're skipping testing for now. I'll make a note in confluence.

Reply:
ok cool